cmd/cluster-operator/scheme/aws.go

```diff
@@ -19,36 +19,29 @@ package scheme
 
 import (
 	cgscheme "k8s.io/client-go/kubernetes/scheme"
-	infrav1alpha3 "sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3"
-	infrav1alpha4 "sigs.k8s.io/cluster-api-provider-aws/api/v1alpha4"
-	infrav1 "sigs.k8s.io/cluster-api-provider-aws/api/v1beta1"
-	eksbootstrapv1alpha3 "sigs.k8s.io/cluster-api-provider-aws/bootstrap/eks/api/v1alpha3"
-	eksbootstrapv1alpha4 "sigs.k8s.io/cluster-api-provider-aws/bootstrap/eks/api/v1alpha4"
```

Comment:
Assume this is removing the conversion between versions, if so do you also remove the that in CRD?

Reply:
`awsProviderVersion := env("AWS_PROVIDER_VERSION", "v2.0.0")`

Reply:
?

Reply:
after capa upgraded to v2.0.0, all crds are re-generated.